survey_form/index.html

```diff
@@ -43,8 +43,37 @@
 
     <div class="tittle"><h1 class="tittle__item">Survey Form</h1></div>
 
-    <form class="form">
-      <p class="form__tittle">Lets us now how we can improve freeCodeCamp</p>
+    <form class="form-survey">
+      <legend class="form__tittle">
+        Lets us now how we can improve freeCodeCamp
+      </legend>
+
+      <label for="name">Name: </label>
+      <input
+        type="text"
+        id="name"
+        class="form-survey__input"
+        required
+        placeholder="Leonardo DiCaprio"
+      />
```

Comment:
Proponuję też w CSSach dodać margin pomiędzy inputem, a labelem do kolejnego inputa, ułatwia to użytkownikowi "skanowanie" formularza, szczególnie rozbudowanego.